ios/src/TGLabelPickResult.h

```diff
@@ -0,0 +1,24 @@
+//
+//  TGLabelPickResult.h
+//  tangram
+//
+//  Created by Karim Naaji on 11/02/16.
+//
+
+#import <Foundation/Foundation.h>
+#import "TGGeoPoint.h"
+
+typedef NS_ENUM(NSInteger, TGLabelType) {
+    TGLabelTypeIcon = 0,
+    TGLabelTypeText,
+};
+
+@interface TGLabelPickResult : NSObject
+
+- (id) initWithCoordinates:(TGGeoPoint)coordinates type:(TGLabelType)type properties:(NSDictionary *)properties;
```

Comment:
You'll want to define whether this return is Nullable or not, because Swift. Looks like you'd want `nonnull` Also, use `instancetype` instead of `id` for safer checking and language features.